src/components/wallet/wallet.component.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+
+import useWalletStyles from './wallet.styles'
+import Title from '../shared/title/title.component'
+import Button from '../shared/button/button.component'
+import Carousel1 from '../../assets/images/wallet/carousel1.inline.svg'
+import Carousel2 from '../../assets/images/wallet/carousel2.inline.svg'
+import Carousel3 from '../../assets/images/wallet/carousel3.inline.svg'
+import Carousel4 from '../../assets/images/wallet/carousel4.inline.svg'
+
+const Wallet = () => {
+  const classes = useWalletStyles()
+
+  return (
+    <>
+      <div className={classes.sectionWrapper}>
+        <section className={`${classes.section} ${classes.section1}`}>
+          <div className={`${classes.row} ${classes.section1Row}`}>
+            <div className={`${classes.col3} ${classes.section1title}`}>
+              <Title>Simple and self-custodial wallet for Hermez Network</Title>
+              <div className={classes.buttonWrapper}>
+                {/* TODO add link to the wallet */}
+                <Button to='' text='Open web wallet' />
```

Comment:
you can probably already link to the testnet one

Reply:
Added